content/en/watchdog/insights.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Watchdog Insights
+kind: documentation
+description: View anomalies and outliers in context with Watchdog Insights
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-watchdog-insights-log-management/"
+  tag: "Blog"
+  text: "Augmented troubleshooting with Watchdog Insights"
+- link: "https://www.datadoghq.com/blog/watchdog-insights-apm/"
+  tag: "Blog"
+  text: "Automatically detect error and latency patterns with Watchdog Insights for APM"
+---
+
+## Overview
+
+Watchdog Insights inform you of anomalies and error outliers that relate to your current view, including any facets or search terms. During an incident, use Watchdog Insights to focus on the highest priority data and speed up investigation.
+
+Each Insight highlights one outlier or anomaly affecting a subset of users. Depending on the product area, Watchdog Insights can detect different types of anomalies.
+
+## Locations within Datadog
+
+Three areas within Datadog feature Watchdog Insights: Log Management, RUM, and APM.
```

Comment:
Should we add Profiling to this, as it's in public beta? Containers is also on for 30% of customers so far. cc @samirbrizini

Reply:
@brimtown Profiling is part of APM. In the APM subsection, I said that Insights are available in Continuous Profiler. I used that name because the blog post also calls it Continuous Profiler: https://app.datadoghq.com/release-notes/detect-common-bottlenecks-and-the-underlying-code-that-causes-them-with-watchdog-insights-in-continuous-profiler. Is there a better name I should use?

Reply:
Ah nope, that makes sense to me!